generators/utils.js

```diff
@@ -94,13 +94,16 @@ function configureGlobalDotnetcore() {
     this.relativeMainTestDir = `${this.relativeMainClientDir}/test`;
     this.testProjectDir = `${this.pascalizedBaseName}${constants.PROJECT_TEST_SUFFIX}`;
     this.clientTestProject = `${this.mainClientDir}/test/`;
+    this.kebabCasedBaseName = _.kebabCase(this.baseName);
```

Comment:
configureGlobalDotnetcore seems to depend only on baseName.
baseName is supposed to be configured at app generator, so it should be settled when composing with others generators.

I recommend to rename to customizeDotnetPaths, and move to initializing priority at every generator.